storage/src/vespa/storage/distributor/distributor_stripe_pool.cpp

```diff
@@ -1,13 +1,15 @@
 // Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
 #include "distributor_stripe_pool.h"
 #include "distributor_stripe_thread.h"
+#include <vespa/storage/common/bucket_stripe_utils.h>
 #include <vespa/vespalib/util/size_literals.h>
 #include <cassert>
 
 namespace storage::distributor {
 
 DistributorStripePool::DistributorStripePool()
     : _thread_pool(512_Ki),
+      _n_stripe_bits(),
```

Comment:
Consider explicitly initializing to 0 since it's a numeric member field

Reply:
Fixed